File: app/Ordered.py
from AbstractEvent import AbstractEvent
import json
from datetime import datetime

class Ordered(AbstractEvent):
    orderid : int
    foodid : str
    qty : str
    status : str
    customerid : str
    address : str
    
    def __init__(self):
        super().__init__()
        self.orderid = None
        self.foodid = None
        self.qty = None
        self.status = None
        self.customerid = None
        self.address = None

File: customer/src/main/java/fooddeliverytest/domain/Ordered.java
package fooddeliverytest.domain;

import fooddeliverytest.domain.*;
import fooddeliverytest.infra.AbstractEvent;
import lombok.*;
import java.util.*;
@Data
@ToString
public class Ordered extends AbstractEvent {

    private Long orderid;
    private String foodid;
    private String qty;
    private String status;
    private String customerid;
    private String address;
}


